app/utils/Navigator.scala

```diff
@@ -191,8 +191,7 @@ class Navigator @Inject extends Logging with FeatureSwitching {
     RegisteringBusinessId -> { userAnswers =>
       userAnswers.registeringBusiness match {
```

Comment:
can just replace the whole match statement with pageIdToPageLoad(RegistrationReasonId)